Jenkinsfile-sbom-release

```diff
@@ -125,34 +137,43 @@ def mergeSpdxComponents(String buildDir, String finalComponentName, String final
     """
 }
 
+def getNexusReportName(String tag) {
+  for(entry in NEXUS3_REPORT_BY_TAG) {
+    if(tag ==~ entry.key) {
+      return entry.value
+    }
+  }
+  return DEFAULT_NEXUS3_REPORT
+}
+
 dockerizedRunPipeline(
     skipVulnerabilityScan: true,
     pathToDockerfile: "./build-images/Dockerfile.sbom-deployer",
     prepare: {
       withSonatypeDockerRegistry() {
-        sh "docker pull sonatype/nexus3:${params.docker_nexus3_tag}"
-        env['nexusVersion'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+        sh "docker pull ${DOCKER_NEXUS_IMAGE_NAME}:${params.docker_nexus3_tag}"
+  
+        def nexusVersion = sh(script: "docker inspect ${DOCKER_NEXUS_IMAGE_NAME}:${params.docker_nexus3_tag} \
             | jq -r '.[0].Config.Labels.version' ",
           returnStdout: true).trim()
-        env['dockerImageVersion'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+        def dockerImageVersion = sh(script: "docker inspect ${DOCKER_NEXUS_IMAGE_NAME}:${params.docker_nexus3_tag} \
             | jq -r '.[0].Config.Labels.release' ",
           returnStdout: true).trim()
-        env['ubiImageId'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+        def ubiImageId = sh(script: "docker inspect ${DOCKER_NEXUS_IMAGE_NAME}:${params.docker_nexus3_tag} \
             | jq -r '.[0].Config.Labels.\"base-image-ref\"' \
             | sed -En 's/^.+image=(.+)\$/\\1/p'",
           returnStdout: true).trim()
+        
+        env['imageTag'] = params.docker_nexus3_tag
+        env['nexusVersion'] = nexusVersion
+        env['dockerImageVersion'] = dockerImageVersion
+        env['ubiImageId'] = ubiImageId
```

Comment:
```suggestion
env['imageTag'] = params.docker_nexus3_tag
        env['nexusVersion'] = dockerInspect('version')
        env['dockerImageVersion'] = dockerInspect('release')
        
        def baseImageRef = dockerInspect('base-image-ref')
        env['ubiImageId'] = baseImageRef.contains("image=") ? baseImageRef.split("image=")[1] : ""

Reply:
Solved in [2e0b43d](https://github.com/sonatype/docker-nexus3/pull/199/commits/2e0b43def3d8bd7bee7d206f0727ec63132ce94c)